test/mocha/mock.data.js

```diff
@@ -622,3 +694,158 @@ mockData.checksIncludeStatusVerificationResultSchema = {
   }
 }
 /* eslint-enable */
+
+/* eslint-disable */
+mockData.envelopedVpRequest = {
+  query: [
+    {
+      type: 'QueryByExample',
+      credentialQuery: {
+        reason: 'Enveloped Verifiable Credential required.',
+        example: {
+          '@context': [
+            'https://www.w3.org/ns/credentials/v2'
+          ],
+          type: [
+            'EnvelopedVerifiableCredential'
```

Comment:
This isn't quite right as the query should be about the contents of the envelope. Asking for an enveloped VC doesn't really say at all what should be in the envelope. I don't think this is actually used in the tests that this was added for, so I'll just turn this into an issue to get it changed after this PR is merged to prevent its holdup.